pkg/auth/db_service.go

```diff
@@ -308,10 +318,16 @@ func (s *DBAuthService) ListUserPolicies(ctx context.Context, username string, p
 		})
 	slice, paginator, err := ListPaged(ctx, s.db, reflect.TypeOf(policy), params, "display_name",
 		psql.Select("*").FromSelect(sub, "p"))
+
 	if slice == nil {
 		return nil, paginator, err
 	}
-	return slice.Interface().([]*model.BasePolicy), paginator, nil
+	policies := slice.Interface().([]*model.DBPolicy)
+	res := make([]*model.BasePolicy, 0, len(policies))
+	for _, p := range policies {
+		res = append(res, &p.BasePolicy)
+	}
+	return res, paginator, nil
```

Comment:
How is this related?

Reply:
This is a bug in `ListUserPolicies` that was not detected as nothing triggers this fuynction. I bumped into it with my migration test, that does use it and had to fix it to get the migration test legit.
Should have done it in a separate commit, but it slipped my mind